src/utils/AuthService.js

```diff
@@ -83,7 +83,7 @@ export default class AuthService extends EventEmitter {
     this.storage.setItem('id_token', idToken);
 
     Cookies.set('authorization', `Bearer ${idToken}`, {
-      domain: `.${window.location.hostname}`
```

Comment:
I don't think this is correct. the `.` prefix means the cookie will be used for subdomains, which is what we want since the request being sent is to `api.thedataloom.com`